packages/stream_video_flutter/lib/src/call_screen/call_content/call_content.dart

```diff
@@ -161,7 +161,7 @@ class _StreamCallContentState extends State<StreamCallContent>
   Widget build(BuildContext context) {
     final theme = StreamVideoTheme.of(context);
 
-    if (_isPictureInPictureModeOn) {
+    if (_isPictureInPictureModeOn && CurrentPlatform.isAndroid) {
```

Comment:
Doesn't this restrict screensharing to iOS?

Reply:
It's for Picture in Picture :) It's not connected to the whole PR it's just something I noticed and fixed in the meantime.